vultr/resource_vultr_load_balancer.go

```diff
@@ -622,3 +650,13 @@ func generateSSL(sslData interface{}) *govultr.SSL {
 		Chain:       config["chain"].(string),
 	}
 }
+
+func generateAutoSSL(autoSSLData interface{}) *govultr.AutoSSL {
+	k := autoSSLData.(*schema.Set).List()
```

Comment:
Why are you casting to list?  This is a struct so shouldn't need to do this.  Refer to how the [health check](https://github.com/vultr/terraform-provider-vultr/blob/master/vultr/resource_vultr_load_balancer.go#L600) is done, for example.